backend/src/main/java/com/example/parkingspot/service/PersonService.java

```diff
@@ -31,11 +31,13 @@ public List<Person> getAllPersons() {
   }
 
   public Person registerNewPerson(Person person) {
-    Optional<Person> existingPerson = personRepository.findByFirstNameAndLastName(person.getFirstName(),
-        person.getLastName());
-    if (existingPerson.isPresent()) {
-      return person;
-    }
+    // . EDIT THIS CHECK FOR OTHER UNIQUE PROP -
```

Comment:
Is this needed? I don't understand "For other unique prop"

Reply:
Future logic is needed, somewhere to check if the Person that is getting registered might already exist. There is no need for that complexity now, for this small app and whitout unique data (ie. socialsecuritynumber)